Networking/Networking/Model/Product/ProductAddOnEnvelope.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+
+/// Helper to unwrap the product add-ons envelope inside the product metadata.
+/// Sample Json:
+/// "meta_data": [
+/// {
+///  "id": 6469,
+///  "key": "_last_editor_used_jetpack",
+///  "value": "classic-editor"
+///},
+///{
+///  "id": 6471,
+///  "key": "_product_addons",
+///  "value": [
+///    {
+///      "description": "Pizza topping",
+///      "title_format": "label",
+///      "required": 0,
+///    }
+///  ]
+///}
+///]
```

Comment:
nit: JSON snippet starts with 1 space left interval (from ///), but then it slips to 0 interval.